src/app.js

```diff
@@ -2,8 +2,16 @@
 'use strict';
 
 const fs = require('fs');
+const args = process.argv.slice(2);
+const [source, destination] = args;
 
 function cp(location1, location2) {
+  if (args.length !== 2) {
+    console.error('Usage: node yourScript.js <source> <destination>');
+
+    return;
+  }
+
```

Comment:
I didn't mean for you to move this condition inside function but rather to add else statement so that if an error occurs there's no need to run the function ;)